.gitignore

```diff
@@ -23,3 +23,6 @@ coverage
 serverless-output.yml
 serverless-output.yaml
 serverless_config.json
+
+.yalc
+yalc.lock
```

Comment:
Add new line at end of file(It makes git happy). 

Usually eslint forces us to add new lines at the end of .ts files, but for other file types you need to manually add it.
You can set up your IDE to always add it https://intellij-support.jetbrains.com/hc/en-us/community/posts/207071445-On-save-new-line-added-to-end-of-file-How-stop-this-.

Reply:
added